### ISSUE TITLE
YOLO resolution bigger then image resolution. Any profit?

### ISSUE THREAD

**Issue by WadRex** [AlexeyAB/darknet#3687]
Hello,
I have one question. If I have an image with resolution 416x416, will I benefit from Yolo with higher resolution? For example 832x832 or 1024x1024. I don't care about speed at this point, just pure accuracy.
UPD: In this comment https://github.com/AlexeyAB/darknet/issues/3114#issuecomment-494148968 there is "yolo_v3_spp_pan_lstm.cfg", but no .cfg or .weights file. Is it still in developement or you just forgot to update this table?

**Comment by LukeAI** [comment]
I think it's unlikely - the image gets resized to the size of the network. best way to find out is to try, tho - let us know if you run an experiment?

**Comment by WadRex** [comment]
@LukeAI Yep, you were right.
It end up getting much worse.
I have only one class in my dataset. I trained it on "Tiny PAN LSTM" model.
On 416x416 on my best weights I got 78% of accuracy.
On 832x832 I got 36%.
It might be that higher resolution requies more training images, but I haven't tested it yet, maybe sometime in the future.
Btw, I got ~5000 image sequences for training and ~1000 for validation.
